### PR TITLE
Popup window portfolio project pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
Added a popup window onthe see project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,143 +60,7 @@ <h3 class="connect">LET’S CONNECT</h3>
...
-                    <p class="works-container-text">A daily selection of privately <br class="must-ignor">
-                        personalized reads;no accounts or<br class="must-ignor">
-                        sign-ups required.</p>
-
-                    <ul class="primarytext3">
-                        <li class="lang">HTML</li>
-                        <li class="lang must-ignor2">Ruby on rails</li>
-                        <li class="lang">CSS</li>
-                        <li class="lang JavaScript">JavaScript</li>
-                    </ul>
```
Comment (ghost) [inline index.html:155]: - [ ] Good work on your design but there are some aspects you can improve on. 
  - Your image's aspect ratio does not appear to be balanced. Kindly adjust the image's height and if the height is too long you can use a scroll bar.
  - Kindly position the close button at the top right corner to match the original design
  - Kindly also position the `tonic` header  correctly to match the original design

image
****

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AleWaweru) [reply index.html:155]: done

**Review by divinecharlotte — APPROVED** [review]
# Hello @AleWaweru     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**